modules/extension/xsd/xsd-wfs/src/main/java/org/geotools/wfs/v1_1/PropertyTypeBinding_1_1.java

```diff
@@ -0,0 +1,87 @@
+package org.geotools.wfs.v1_1;
```

Comment:
This file is missing the header

```suggestion
/*
 *    GeoTools - The Open Source Java GIS Toolkit
 *    http://geotools.org
 *
 *    (C) 2024, Open Source Geospatial Foundation (OSGeo)
 *
 *    This library is free software; you can redistribute it and/or
 *    modify it under the terms of the GNU Lesser General Public
 *    License as published by the Free Software Foundation;
 *    version 2.1 of the License.
 *
 *    This library is distributed in the hope that it will be useful,
 *    but WITHOUT ANY WARRANTY; without even the implied warranty of
 *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 *    Lesser General Public License for more details.
 */

package org.geotools.wfs.v1_1;
```